packages/react-ui/components/Button/Button.tsx

```diff
@@ -278,28 +279,26 @@ export class Button extends React.Component<ButtonProps, ButtonState> {
           {arrow}
           <div className={jsStyles.caption()}>
             {icon}
-            {this.props.children}
+            <span className={cn({ [jsStyles.visibilityHidden()]: loading && !icon })}>{this.props.children}</span>
```

Comment:
```suggestion
            <span className={cn({ [jsStyles.visibilityHidden()]: loading })}>{this.props.children}</span>
```

`&& !icon` ведь больше тоже не нужен.